src/QueryBuilder/AbstractDDLQueryBuilder.php

```diff
@@ -169,14 +169,15 @@ public function createTable(string $table, array $columns, string $options = nul
     {
         $cols = [];
 
-        /** @psalm-var string[] $columns */
+        /** @psalm-var array<string, ColumnInterface>|string[] $columns */
```

Comment:
```suggestion
        /** @psalm-var array<array-key, ColumnInterface|string> $columns */
```

Reply:
If column is `ColumnInterface` it should have name as an array key => `array<string, ColumnInterface>`
If it is `string` then the array key can be `int|string` => `string[]`

Reply:
Can you add columns format to phpdoc in `DDLQueryBuilderInterface`?